awx/ui_next/src/screens/Credential/CredentialDetail/CredentialDetail.jsx

```diff
@@ -0,0 +1,172 @@
+import ContentLoading from '@components/ContentLoading';
+import DeleteButton from '@components/DeleteButton';
+import { DetailList, Detail, UserDateDetail } from '@components/DetailList';
+import ErrorDetail from '@components/ErrorDetail';
+import { CredentialsAPI, CredentialTypesAPI } from '@api';
+import { Credential } from '@types';
+
+function CredentialDetail({ i18n, credential }) {
+  const {
+    id: credentialId,
+    name,
+    description,
+    inputs,
+    created,
+    modified,
+    summary_fields: {
+      credential_type,
+      organization,
+      created_by,
+      modified_by,
+      user_capabilities,
+    },
+  } = credential;
+
+  const [fields, setFields] = useState([]);
+  const [contentError, setContentError] = useState(null);
+  const [deletionError, setDeletionError] = useState(null);
+  const [hasContentLoading, setHasContentLoading] = useState(true);
+  const history = useHistory();
+
+  useEffect(() => {
+    (async () => {
+      setContentError(null);
+      setHasContentLoading(true);
+      try {
+        const {
+          data: { inputs: credentialTypeInputs },
+        } = await CredentialTypesAPI.readDetail(credential_type.id);
+
+        setFields(credentialTypeInputs.fields);
+      } catch (error) {
+        setContentError(error);
+      } finally {
+        setHasContentLoading(false);
+      }
+    })();
+  }, [credential_type.id]);
+
+  const handleDelete = async () => {
+    setHasContentLoading(true);
+
+    try {
+      await CredentialsAPI.destroy(credentialId);
+      history.push('/credentials');
+    } catch (error) {
+      setDeletionError(error);
+    }
+    setHasContentLoading(false);
+  };
+
+  const renderDetail = ({ id, label, type, secret }) => {
+    let detail;
+
+    if (type === 'boolean') {
+      detail = (
+        <Detail
+          key={id}
+          label={i18n._(t`Options`)}
+          value={<List>{inputs[id] && <ListItem>{label}</ListItem>}</List>}
+        />
+      );
+    } else if (secret === true) {
+      detail = <Detail key={id} label={label} value={i18n._(t`Encrypted`)} />;
+    } else {
+      detail = <Detail key={id} label={label} value={inputs[id]} />;
+    }
+
+    return detail;
+  };
+
+  if (hasContentLoading) {
+    return <ContentLoading />;
+  }
+
+  if (contentError) {
+    return <ContentError error={contentError} />;
+  }
+
+  return (
+    <CardBody>
+      <DetailList>
+        <Detail label={i18n._(t`Name`)} value={name} />
+        <Detail label={i18n._(t`Description`)} value={description} />
+        {organization && (
+          <Detail
+            label={i18n._(t`Organization`)}
+            value={
+              <Link to={`/organizations/${organization.id}/details`}>
+                {organization.name}
+              </Link>
+            }
+          />
+        )}
+        <Detail
+          label={i18n._(t`Credential Type`)}
+          value={
+            <Link to={`/credential_types/${credential_type.id}/details`}>
```

Comment:
Only user-created credential types will have details. [Managed credential types](https://github.com/ansible/awx/blob/devel/awx/main/models/credential/__init__.py#L591-L1156) aren't shown in the UI.

Credential types have a `managed_by_tower` field which we'd probably want to use but it's not currently on the `credential.summary_fields.credential_type` object.